Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -365,10 +341,48 @@ def get_file_request(self, sha256: List = None, expiration_seconds: int = None)
         return self._http_request('POST', suffix_url, json_data=body)
 
     def get_file_path_request(self, sha256: str = None) -> Dict:
-
         suffix_url = f'/ubs/v1/orgs/{self.cb_org_key}/sha256/{sha256}/summary/file_path'
         return self._http_request('GET', suffix_url)
 
+    def create_search_process_request(self, process_hash: str, process_name: str, event_id: str, query: str,
+                                      limit: int = None) -> dict:
+        if not process_hash and not process_name and not event_id and not query:
```

Comment:
can a use input `query` and 1 or more of the others? if not - please add a validation and a ut for the failed case

Reply:
yes, a user is alowed to put both query and at lease one of the parameters.